cli/src/main/java/tools/vitruv/cli/options/MetamodelOption.java

```diff
@@ -25,6 +25,7 @@
 public class MetamodelOption extends VitruvCLIOption {
   // resource/tools.vitruv.methodologisttemplate.model/src/main/ecore/model.genmodel
   public static final String SUBFOLDER = "/model/src/main/ecore/";
+  private String modelName;
 
   public MetamodelOption() {
```

Comment:
Just to understand, we want to create view types for every metamodel, every time.
There is/should not be an extra option to generate identity view types.
Do I have this right?